README.md

```diff
@@ -54,13 +53,12 @@ To install the FortNerf App Follow these steps
 ### Prerequisites
 <a name="prerequisites"></a>
 * npm
-  ```sh
+  ```
   npm install npm@latest -g
   ```
 
 ### Installation
 <a name="installation"></a>
-_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
 
 1. Clone the repo
   ```she
```

Comment:
this is what i meant by the 'she'

Reply:
but i guess its jus a 'best practice' kinda thing having the beginning line as ```sh or ```bash, not really a necessity. jus make note of it and change it on a next commit or something since the reader shouldn't be able to see this unless they look at a raw version of your readme.